content/en/docs/languages/go/generated-code.md

```diff
@@ -3,12 +3,12 @@ title: Generated-code reference
 linkTitle: Generated code
 weight: 95
 ---
-
-This page describes the code generated with the [grpc plugin](https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc), `protoc-gen-go-grpc`,
-when compiling `.proto` files with `protoc`.
+This page describes the code generated when compiling `.proto` files with `protoc`, using the `protoc-gen-go-grpc` [grpc plugin](https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc). This latest version of generated code uses generics by default. If you're working with older generated code that doesn't use generics, you can find the relevant documentation [here](/docs/languages/go/generated-code-old).  While we encourage using this latest version with generics, you can temporarily revert to the old behavior by setting the `useGenericStreams` flag to `false`.
```

Comment:
Please update the issue about removing this flag to include some notes that we need to update this documentation when the flag is removed.

Reply:
Noted. Will make the changes on the issue.